### ISSUE TITLE
Can't install latest version on Versa 2

### ISSUE THREAD

**Issue by Schatteneis** [jdhartley/terminal-for-fitbit#45]
i get an error saying the app was built for atlas and Vulcan but device only supports mira

**Comment by jdhartley** [comment]
Hi @Schatteneis, for Versa 2 please use the `sdk4` branch. Right now the latest version for mira devices is v2.0.2

**Comment by jdhartley** [comment]
Hi @Schatteneis, I have backported as many features from the latest Terminal SDK5 branch to the SDK4 branch, https://github.com/jdhartley/terminal-for-fitbit/commits/sdk4.

This commit should work on your Versa 2 / mira. 8c756702138b5678d991c90acfb3e48a16dcfb4c

I have tagged and added 2.3.0-legacy to the Releases here: https://github.com/jdhartley/terminal-for-fitbit/releases/tag/2.3.0-legacy. I will also submit it to the Fitbit developer portal today and the update should be available in the coming days. Hope you enjoy!
